# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wings3d/wings.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ur wings-0.98.26b_orig/plugins_src/accel/Makefile wings-0.98.26b/plugins_src/accel/Makefile
--- wings-0.98.26b_orig/plugins_src/accel/Makefile	2004-11-27 20:27:16.000000000 +0100
+++ wings-0.98.26b/plugins_src/accel/Makefile	2005-04-22 19:39:47.408000400 +0200
@@ -33,7 +33,7 @@
    GL_LIBS =
  else
    LIBS = -shared -fpic
-   GL_LIBS = -lGLU -lGL
+   GL_LIBS = -lGLU -lGL -L/usr/X11/lib
  endif
 endif
 
